### ISSUE TITLE
make more generic eslint config with no stylistic defaults

### ISSUE THREAD

**Issue by jedwards1211** [jedwards1211/meteor-webpack-react#31]
@AdamBrodzinski I didn't notice until today that you snuck in the airbnb eslint config :)  Since it's extremely strict about whitespace, dangling commas, and other stylistic things, I'd rather replace it with a more generic eslint config at some point, basically just `eslint:recommended` with maybe a few tweaks.

**Comment by AdamBrodzinski** [comment]
lol, yea I though I mentioned that :laughing: A less strict one would be totally fine. The default eslint won't lint ES6/7 so i'm sure there's one that will work.

**Comment by AdamBrodzinski** [comment]
Threre's also a way to run eslint through webpack so that it will fail the build on an error. Not sure if this is better or not. We would still need the `.eslintrc` for editors though.

**Comment by jedwards1211** [comment]
Yeah i have that working in another project, so eventually I could port it to this!
